phpunit.xml.dist.11

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://phpunit.de/manual/4.1/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
```

Comment:
```suggestion
         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

maybe there is more to take from https://github.com/symfony/recipes/blob/main/phpunit/phpunit/11.1/phpunit.dist.xml